pkgs/applications/video/openshot-qt/libopenshot-audio.nix

```diff
@@ -28,6 +28,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-XtwTZsj/L/sw/28E7Qr5UyghGlBFFXvbmZLGXBB8vg0=";
   };
 
+  # https://github.com/OpenShot/libopenshot-audio/issues/112
```

Comment:
That issue doesn't provide too much info, is there a better description?

Reply:
Indeed, it would be better to write a `.patch` file with a header describing it.

And, possibly, send it to upstream developers.

Reply:
I believe https://forum.juce.com/t/juce-and-macos-11-arm/40285/24 gives a better description. Should I leave this link as is?